index.html

```diff
@@ -5,8 +5,56 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;600&display=swap" rel="stylesheet">
+
+    
 </head>
 <body>
     
+    <div class="container">
+        <div class="header"><h1>Header</h1></div>
+
+
+
+        <div class="temp" onclick="tempColor()"><h2>Temp</h2>
+        
+            
+            <button id="increment">⬆️</button>
+            <button id="decrement">⬇️</button>
+            
+            <temperature id="temp"> 
```

Comment:
`temperature` isn't a valid html element type?

Instead maybe make it a `section` and then give it the class `temperature`